src/CoreBundle/Resources/views/snippets-cms/NavigationTree/standard.html.twig

```diff
@@ -0,0 +1,84 @@
+{# @var treeNodes \ChameleonSystem\CoreBundle\DataModel\BackendTreeNodeDataModel #}
+
+{% if isInIframe == false %}
+
+<div class="card">
+    <div class="card-header">
+        <h3>{{ 'chameleon_system_core.cms_module_page_tree.headline' | trans }}</h3>
+    </div>
+    <div class="card-body simple-tree-card">
+{% endif %}
+        <div class="row m-0">
+            <div class="col-12 col-md-6 col-xl-8">
+                <div id="navigationTreeDataContainer"
+                     data-current-page-id = "{{ currentPageId }}"
+                     data-tree-nodes-ajax-url = "{{ treeNodesAjaxUrl | raw }}"
```

Comment:
Does this work with e("html_attr") instead of raw or nothing?

Reply:
"currentPageId" isn't set and isn't needed here, it's part of the Url. So I removed currentPageId.
The Urls are built with the getArrayAsUrl method, which already does URL-encoding, so "raw" is correct here.